Extensions/ShareTo/SendToDevice.swift

```diff
@@ -2,31 +2,34 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0
 
-import UIKit
 import Shared
 import Storage
 import SwiftUI
+import UIKit
 
 class SendToDevice: DevicePickerViewControllerDelegate, InstructionsViewDelegate {
 
     var sharedItem: ShareItem?
     weak var delegate: ShareControllerDelegate?
+    private var profile: Profile {
+        return BrowserProfile(localName: "profile")
+    }
```

Comment:
I think we don't need to create a profile here, since in DevicePickerViewController there's the following:
```
    fileprivate func ensureOpenProfile() -> Profile {
        // If we were not given a profile, open the default profile. This happens in case we are called from an app
        // extension. That also means that we need to shut down the profile, otherwise the app extension will be
        // terminated when it goes into the background.
```
So we should pass no profile from the extension to the SendToDeviceHelper, and the DevicePickerViewController will handle on it's on already

Reply:
We need the profile on `SendToDevice` for one of the `DevicePickerViewController` delegates to call to sendItem and we also need the profile on `SendToDeviceHelper` to determine if the user hasAccount to show the Instructions or the DevicePicker